client/src/util/stateManager/universe.js

```diff
@@ -118,7 +81,7 @@ function reconcileSchemaCategoriesWithSummary(universe) {
     ) {
       const categories = _.union(
         _.get(s, "categories", []),
-        _.get(universe.summary.obs[s.name], "categories", [])
+        _.get(universe.obsAnnotations.col(s.name).summarize(), "categories", [])
```

Comment:
I was a little thrown that summarize was added in the last PR...

Reply:
yes, I need to be better about my PR factoring....